core/cloudflow-cli/src/main/scala/akka/cli/cloudflow/execution/ConfigurationExecution.scala

```diff
@@ -16,7 +16,7 @@ final case class ConfigurationExecution(c: Configuration, client: KubeClient, lo
   def run(): Try[ConfigurationResult] = {
     logger.info("Executing command Configuration")
     for {
-      _ <- validateProtocolVersion(client)
+      _ <- validateProtocolVersion(client, c.namespace)
```

Comment:
Here you are assuming the protocol version configmap exists in the namespace where the Cloudflow application is deployed.
This is (in most cases) not correct, since the configmap with the protocol version is in the namespace where the operator lives in.

This feature requires a little design (we discussed previously with @RayRoestenburg ), some options are:
- The operator installs the protocol version in the target namespace,
- We include (somehow) this information in the CRD,
- We provide an additional option to the CLI "operatorNamespace" to let the user specify it,
- ...

Reply:
I missed the conversation with @RayRoestenburg XD it would be nice to explore the CRD option, but I won't oppose to use the additional flag 👍 😅